executor/simple.go

```diff
@@ -1262,6 +1262,9 @@ func asyncDelayShutdown(p *os.Process, delay time.Duration) {
 }
 
 func (e *SimpleExec) executeCreateStatistics(s *ast.CreateStatisticsStmt) (err error) {
+	if !e.ctx.GetSessionVars().EnableExtendedStats {
+		return dbterror.ClassOptimizer.NewStd(mysql.ErrInternal).GenWithStack("Extended statistics feature is not generally available now, and tidb_enable_extended_stats is OFF")
```

Comment:
Can we extract this error to planner/core/errors.go

Reply:
IMHO, this error is just for temporary usage, and it would be removed once extended statistics is stable, so we it is unnecessary to do the extract?

Reply:
ok, please fix the ci